client/my-sites/hosting/github/deployment-card/use-deployment-status.ts

```diff
@@ -0,0 +1,36 @@
+import { useQuery, UseQueryOptions } from 'react-query';
+import wp from 'calypso/lib/wp';
+
+export type DeploymentData = {
+	status: string;
```

Comment:
We can use a union here

```suggestion
	status: 'failed' | 'running' | 'success';
```